app/views/components/blockEditor.html

```diff
@@ -21,7 +21,7 @@
   </div>
   <label>{{block.title}} <em class="required" ng-show="block.required">*</em></label>
   <ng-switch on="block.type">
-    <p ng-switch-when="paragraphContent" ng-bind="block.content" class="display-linebreaks"></p>
+    <p ng-switch-when="paragraphContent" ng-bind="block.content.paragraph" class="display-linebreaks"></p>
```

Comment:
Will this affect blocks already stored in the database? Seems like the content would still be at `block.content` when loading them. One of my conferences still seems broken...